src/editing_opr/apply_opr.py

```diff
@@ -0,0 +1,21 @@
+from src.editing_opr.image_opr import ImageOperations
```

Comment:
Editing operation? Please remove the upstream files that are not being used.

Reply:
@gaurav274 These changes come from the frame-storage branch. The catalog module is built over the work of frame-storage branch so we cant remove this unless that branch gets merged to master. Let us know what is the plan for this.

Reply:
@gaurav274 The changes come from the frame-storage branch. The catalog module is built over the work done in the frame-storage branch. We cant remove these files unless the frame-storage branch gets merged to master. The diff will automatically remove these files from the PR then.Let us know the action on this.